python/graphstorm/model/lm_embed.py

```diff
@@ -262,15 +262,17 @@ def _get_model_name(self, ntype):
     def _load_embeddings(self):
         """ Load LM embeddings from files.
         """
+        embed_ndata_names = self.embed_ndata_name
         for ntype in self._lm_models.ntypes:
             embed_path = os.path.join(os.path.join(self._embed_path, ntype),
                     self._get_model_name(ntype))
             if os.path.exists(embed_path):
                 if get_rank() == 0:
                     logging.info("load LM embedding from %s for node type %s",
                             embed_path, ntype)
+                embed_name = embed_ndata_names[ntype]
                 self._lm_emb_cache[ntype] = load_pytorch_embedding(embed_path,
-                        self._g.get_node_partition_policy(ntype), "bert_emb")
+                        self._g.get_node_partition_policy(ntype), embed_name)
```

Comment:
It seems it is still using "bert_emb". Any specific reason we what to define the embed_ndata_name?

Reply:
i think it's better to not hard code the embedding name in multiple places.